recipes/dmff/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "dmff" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/deepmodeling/DMFF/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: d990516e2dc9ba45d87a8a93fc6fc12bc542adf555bba3e706a41e4db1655d71
+
+build:
+  number: 0
+  noarch: python
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - setuptools
+    - setuptools_scm
+  run:
+    - python >=3.8
+    - numpy >=1.18
+    - jax
+    - jaxlib
+    - jax-md >=0.1.28
+    - mdtraj
+    - optax
+    - pymbar
+    - openmm >=7.6.0
+    - rdkit
+    - freud
```

Comment:
Not all of these are listed upstream, are you sure they are required?

Ref: https://github.com/deepmodeling/DMFF/blob/6a7ec8b9e91adfd9c4b5fe4c2ca66d89d2963215/setup.py#L21

Reply:
I followed the [installation documentation](https://github.com/deepmodeling/DMFF/blob/master/docs/user_guide/installation.md). I manually checked each dependence, and removed `jaxlib` since it is already a dependency of `jax`. Others can be found in the source code. (It seems an issue in the upstream that some dependencies are not listed in `setup.py`.)